### PR TITLE
Scripting

### PR DESCRIPTION
**Pull request by benedictnkosi**

### CODE REVIEW

File: src/test/resources/Room.feature [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Feature: Confirming the functionality of the room top column
+
+  Scenario Outline: Validating the functionality of 'Room' column
+    Given The user has logged in the website
+    When The user clicks on the "Rooms" top column
```
Comment (benedictnkosi) [inline src/test/resources/Room.feature:5]: change to menu not column

File: src/test/resources/Room.feature [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    And The user clicks on the Click to Select Dates tab
```
Comment (benedictnkosi) [inline src/test/resources/Room.feature:6]: clicks on the click?

File: src/test/resources/Room.feature [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    Then User validates calendar is present
```
Comment (benedictnkosi) [inline src/test/resources/Room.feature:7]: validates that the calendar is

File: src/test/resources/Room.feature [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+    When User selects the start date and end date
```
Comment (benedictnkosi) [inline src/test/resources/Room.feature:9]: this step must take in parameters so we can change the dates on the feature file

File: src/test/resources/Room.feature [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+    And The user clicks the 'Apply' button
+    Then User validates the dates on input tab are start date and end date
```
Comment (benedictnkosi) [inline src/test/resources/Room.feature:12]: same as above comment

File: src/test/resources/Room.feature [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+    When User clicks footer icon with value 'phone'
```
Comment (benedictnkosi) [inline src/test/resources/Room.feature:14]: this must be a different test with its own feature file. the footer is on all pages its not a room's plage feature

**Review by benedictnkosi — COMMENTED** [review]
Please see comments

File: src/test/resources/Room.feature (hunk 1 above)
Comment (MlulekiN) [reply src/test/resources/Room.feature:5]: Done

File: src/test/resources/Room.feature (hunk 2 above)
Comment (MlulekiN) [reply src/test/resources/Room.feature:6]: Added asterisks to clarify as that is the tab name

File: src/test/resources/Room.feature (hunk 3 above)
Comment (MlulekiN) [reply src/test/resources/Room.feature:7]: Done

File: src/test/resources/Room.feature (hunk 4 above)
Comment (MlulekiN) [reply src/test/resources/Room.feature:9]: Done

File: src/test/resources/Room.feature (hunk 5 above)
Comment (MlulekiN) [reply src/test/resources/Room.feature:12]: Done

File: src/test/resources/Room.feature (hunk 6 above)
Comment (MlulekiN) [reply src/test/resources/Room.feature:14]: Noted

**Pull request closed without merging**